index.html

```diff
@@ -49,5 +47,183 @@ <h1 class="intro__heading">
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
+             </figcaption>
+        </figure>
+
+
+        <figure class="card">
+            <figcaption class="card__img--cont">
+                <img src="./images/SnapshootPortfolio3.png" alt="image of a project" class="img__card">
+                <H2 class="desc__project">Multi-Post Stories</H2>
+              <ul class="project__info--cont">
+                <li class="project__info--title">canopy</li>
+              <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                <li class="project__info--date">2015</li>
+              </ul>
+            <p class="project__desc--text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list__lng--btn">
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">html</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">css</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">javascript</button></li>
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
+             </figcaption>
+        </figure>
+
+    </section>
+    
+    <section class="about">
+     <div class="intro intro__resume">
+            <h1 class="intro__heading">
+                About Myself
+            </h1>
+            <p class="intro__text">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate to contact me.</p>
+             <a href="#" class="intro__contact">let's connect</a>
+            <ul class="intro__social--cont">
+                
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-github-square"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-linkedin-in"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="far fa-hand-peace"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-twitter"></i> </a>   </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-medium-m"></i>   </a>   </li>
+
+            </ul>
+             
+            <button type="button" class="link__btn--resume">Get my resume</button>
```

Comment:
- [ ] Your `Get my resume` button should also have stylings for `:active` and `:hover` effects. Like so:

```
.link__btn--resume:active {
  ...
}

.link__btn--resume:hover {
 ...
}
```